oraclepaas/resource_mysql_service_instance.go

```diff
@@ -0,0 +1,662 @@
+package oraclepaas
+
+import (
+	"fmt"
+	"log"
+	"strconv"
+	"time"
+
+	opcClient "github.com/hashicorp/go-oracle-terraform/client"
+	"github.com/hashicorp/go-oracle-terraform/mysql"
+	"github.com/hashicorp/terraform/helper/schema"
+	"github.com/hashicorp/terraform/helper/validation"
+)
+
+func resourceOraclePAASMySQLServiceInstance() *schema.Resource {
+	return &schema.Resource{
+		Create: resourceOraclePAASMySQLServiceInstanceCreate,
+		Read:   resourceOraclePAASMySQLServiceInstanceRead,
+		Update: resourceOraclePAASMySQLServiceInstanceUpdate,
+		Delete: resourceOraclePAASMySQLServiceInstanceDelete,
+
+		Timeouts: &schema.ResourceTimeout{
+			Create: schema.DefaultTimeout(120 * time.Minute),
+			Delete: schema.DefaultTimeout(120 * time.Minute),
+		},
+
+		Schema: map[string]*schema.Schema{
+
+			"service_name": {
```

Comment:
Thoughts on refactoring this to just `name`? I'd like to keep it consistent it with the other resources in this provider

Reply:
Renamed this (and a few other attributes) to for consistency.